README.md

```diff
@@ -89,7 +89,7 @@ public class MainApplication extends Application implements ReactApplication {
 
 1. Run `npm install react-native-mail --save`
 2. Open your project in XCode, right click on `Libraries` and click `Add
-   Files to "Your Project Name"` [(Screenshot)](http://url.brentvatne.ca/jQp8) then [(Screenshot)](http://url.brentvatne.ca/1gqUD).
+   Files to "Your Project Name"` [(Screenshot)](http://url.brentvatne.ca/jQp8) then [(Screenshot)](https://github.com/pedramsaleh/react-native-mail/blob/master/add-xcodeproj.png?raw=true).
```

Comment:
linking it against this repo would help maintain it well in future. It's a major concern so i will just leave it as is unless you want to send a PR :)